src/frontend/src/catalog/schema_catalog.rs

```diff
@@ -30,12 +30,13 @@ use crate::catalog::source_catalog::SourceCatalog;
 use crate::catalog::system_catalog::SystemTableCatalog;
 use crate::catalog::table_catalog::TableCatalog;
 use crate::catalog::view_catalog::ViewCatalog;
-use crate::catalog::{ConnectionId, SchemaId, SinkId, SourceId, ViewId};
+use crate::catalog::{ConnectionId, DatabaseId, SchemaId, SinkId, SourceId, ViewId};
 
 #[derive(Clone, Debug)]
 pub struct SchemaCatalog {
     id: SchemaId,
     name: String,
+    database_id: DatabaseId,
```

Comment:
To my understanding, this new field is added in #11005 for `rw_schemas` so that it can join `rw_databases` in the view definition of `information_schema.tables`. But the join is actually unnecessary https://github.com/risingwavelabs/risingwave/pull/11005/files#r1271849626

Without that case, this can still be viewed as a bug fix because it is a common need to find the database of a schema.

Does this capture the rationale behind this change?

Reply:
Yes, I think the database of a schema will be a common need somewhere in the future. So let's keep it.